Sim/doc/FullSimulation.md

```diff
@@ -200,6 +200,8 @@ Any communication with the Geant Run Manager is handled by this service:
   - [passing an event to `G4EventManager`](#event-processing)
   - [retrieving a simulated event (with hits collections and other information)](#output)
 
+Additionally, the simulation service takes the random number generator seeds from Gaudi service `RndmGenSvc`. If user wants to set the simulation seeds manually, flag `rndmFromGaudi` needs to be set to `false`.
```

Comment:
if *a* user wants..., *the* flag...
and maybe it is better to spell out "randomNumbersFromGaudi" for the parameter